mail/gnupg2/Portfile

```diff
@@ -4,14 +4,14 @@ PortSystem          1.0
 
 set my_name         gnupg
 name                ${my_name}2
-version             2.4.3
+version             2.4.4
 revision            1
```

Comment:
revision should be reset to zero when updating the version

Reply:
Done